compute/compute-node.Dockerfile

```diff
@@ -1463,6 +1463,73 @@ WORKDIR /ext-src/pg_repack-src
 RUN make -j $(getconf _NPROCESSORS_ONLN) && \
     make -j $(getconf _NPROCESSORS_ONLN) install
 
+
+#########################################################################################
+#
+# Layer "pgaudit"
+# compile pgaudit extension
+#
+#########################################################################################
+
+FROM build-deps AS pgaudit-src
+ARG PG_VERSION
+WORKDIR /ext-src
+RUN case "${PG_VERSION}" in \
+    "v14") \
+    export PGAUDIT_VERSION=1.6.2 \
+    export PGAUDIT_CHECKSUM=1f350d70a0cbf488c0f2b485e3a5c9b11f78ad9e3cbb95ef6904afa1eb3187eb \
```

Comment:
Ugh, so if I read it right, their new versions do not have backward compatibility? https://github.com/pgaudit/pgaudit?tab=readme-ov-file#postgresql-version-compatibility

Reply:
In my understanding, they are backward compatible within one major version.
Each major version has separate branch, but it doesn't change anything for our use-case